src/app/static/src/js/components/workflowWizard/workflowWizard.vue

```diff
@@ -0,0 +1,13 @@
+<template>
```

Comment:
Not sure this component is doing much? It feels like runWorkflow is the actual wizard - it manages the logical connections between steps and how to get between them. Maybe rename that to runWorkflowWizard and get rid of this?